packages/web-pkg/src/composables/keyboardActions/useKeyboardActions.ts

```diff
@@ -0,0 +1,75 @@
+import { onBeforeUnmount, onMounted, ref } from 'vue'
+
+export enum Key {
+  C = 'c',
+  V = 'v',
+  A = 'a',
+  Space = ' ',
+  ArrowUp = 'ArrowUp',
+  ArrowDown = 'ArrowDown',
+  Esc = 'Escape'
+}
+
+export enum ModifierKey {
+  Ctrl = 'Control',
+  Shift = 'Shift'
+}
```

Comment:
Here's and issue, keyCode is considered [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode).
Since this is a re-imagination of keyboard actions, why not just use the `key` property in terms of longevity?

Reply:
As I already wrote you in chat - you are right, `keyCode` is indeed deprecated. Hence it doesn't make sense for us to keep that alive.